pkg/dataset/run.go

```diff
@@ -27,15 +27,20 @@ import (
 	"regexp"
 	"strings"
 
-	common "github.com/commonjava/indy-tests/pkg/common"
+	"github.com/commonjava/indy-tests/pkg/common"
 )
 
 type Info struct {
-	PncBaseUrl   string `json:"pncBaseUrl"`
-	GroupBuildId string `json:"groupBuildId"`
+	PncBaseUrl     string `json:"pncBaseUrl"`
+	BuildId        string `json:"buildId"`
+	BuildType      string `json:"buildType"`
+	TemporaryBuild bool   `json:"temporaryBuild"`
 }
 
 const DATASET_DIR = "dataset"
+const DA_JSON = "da.json"
+const TRACKING_JSON = "tracking.json"
+const INFO_JSON = "info.json"
```

Comment:
A nitpick: we can use multi const declare here:
const(
DATASET_DIR = "dataset"
DA_JSON = "da.json"
TRACKING_JSON = "tracking.json"
INFO_JSON = "info.json"
)